kzz7yh-f06866/cod-ve09v2_kzz7yh-f06866.xml

```diff
@@ -70,6 +70,7 @@
    </p>
             <div xml:id="kzz7yh-f06866-Dd1e118">
                <head xml:id="kzz7yh-f06866-Hd1e120">Quaestio 1</head>
+               <head type="question-title">angeli sint intpere<!--?--> </head>
```

Comment:
This is a good example of where you should jump to the beginning of the Article to see the first question listed as a question. It is often clearer there.

```suggestion
               <head type="question-title">Utrum angeli sint in tempore vel in evo</head>
```